<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="percentButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(160,160,160);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="arrowButton">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>110</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(160,160,160);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divineButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>110</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(246,154,6);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(160,160,160);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nineButton">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>197</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiplyButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>197</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(246,154,6);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eightButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>197</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sevenButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>197</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sixButton">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>284</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>284</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(246,154,6);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>284</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>284</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threeButton">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>371</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>371</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(246,154,6);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twoButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>371</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>371</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decimalButton">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>458</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>458</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(246,154,6);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>458</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ansButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>458</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:none;
	border-radius:40px;
	background-color:rgb(49,49,49);
	color:rgb(255,255,255);
	min-width:80px;
	min-height:80px;
	max-width:80px;
	max-height:80px;
	padding:0;

}</string>
    </property>
    <property name="text">
     <string>Ans</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
